frontend/client_sdk/src/CodaSDK.d.ts

```diff
@@ -45,6 +45,15 @@ export declare const genKeys: () => keypair;
   * @returns A public key
   */
 export declare const derivePublicKey: (privateKey: privateKey) => publicKey;
+/**
```

Comment:
should this mention that an exception is raised if the keypair is invalid?

Reply:
good catch, updated!